cli.ts

```diff
@@ -430,59 +426,61 @@ const initProject = (
 		map(_ => i18n.__('bootstrapMsg')),
 	);
 
+const runScaffoldPostInit = (scaffoldDir: SanitizedAbsPath.t): Future<TaqError.t, SanitizedAbsPath.t> => {
```

Comment:
What is PostInit?

Reply:
PostInit is the name of the script in scaffold.json that should be run post-init.